pkg/controllers/secretsync/secret_sync_controller.go

```diff
@@ -103,13 +102,11 @@ func (r *UserDataSecretController) syncSecretData(ctx context.Context, source *c
 		return fmt.Errorf("source secret does not have user data")
 	}
 
-	formatEncoded := base64.StdEncoding.EncodeToString([]byte("ignition"))
```

Comment:
Should it not be base64 encoded? Or was this a double base64 encoding?

Reply:
This was a case of double base64 encoding. What is there currently is correct.